src/backend/api/reservations.js

```diff
@@ -36,7 +36,7 @@ router.post('/', async (req, res) => {
 router.get('/:id', async (req, res) => {
   try {
     const reservationId = req.params.id;
-    const reservation = await knex('reservation').where({ id: reservationId });
+    const reservation = await knex('reservation').where({ meal_id: reservationId });
```

Comment:
Not sure about this one. 
Is meal_id =  reservationId? 🤔

Reply:
Yeah this was hint from the classmate because i couldnt get to work my reservation form but that was not the problem at the end.